csrc/flash_attn/src/flash_fwd_kernel.h

```diff
@@ -1342,23 +1342,24 @@ __forceinline__ __device__ void compute_attn_1rowblock_flashmask(const Params &p
         // for rows outside actual_seqlen_k. So those rows could have Inf / NaN, and the matmul
         // can produce Inf / NaN.
         if (!Is_causal) {
-          if (true/*Is_flashmask*/ && (!enable_mask_bypass ||
-                (m_block + 1) * kBlockM >= gSparseMaskDownMin[n_block] ||
-                m_block * kBlockM < gSparseMaskUpMax[n_block])) {
+          if (true/*Is_flashmask*/ && (!enable_mask_bypass || 
```

Comment:
remove the boolean literal `true`, or add following definition: `constexpr bool Is_flashmask = true;`